js/ui/drop_down_editor/ui.drop_down_list.js

```diff
@@ -369,13 +369,34 @@ var DropDownList = DropDownEditor.inherit({
 
     _createPopup: function() {
         this.callBase();
+        this._findBoundsContainer();
         this._popup._wrapper().addClass(this._popupWrapperClass());
 
         var $popupContent = this._popup.$content();
         eventsEngine.off($popupContent, "mouseup");
         eventsEngine.on($popupContent, "mouseup", this._saveFocusOnWidget.bind(this));
     },
 
+    _findBoundsContainer: function() {
+        var that = this;
+        var containerValue = this.option("dropDownOptions.container");
+        var $container = containerValue && $(containerValue);
+
+        if($container && !typeUtils.isWindow($container.get(0))) {
+            var $containerWithParents = [].slice.call($container.parents());
+            $containerWithParents.unshift($container.get(0));
+
+            each($containerWithParents, function(i, parent) {
+                if(parent === $("body").get(0)) {
+                    return false;
+                } else if(window.getComputedStyle(parent).overflowY === "hidden") {
+                    that._$boundsContainer = $(parent);
+                    return false;
+                }
+            });
```

Comment:
```suggestion
            }.bind(this));
```
And you can remove `that`